src/main/scala/it/pagopa/interop/purposeprocess/api/converters/purposemanagement/PurposeConverter.scala

```diff
@@ -6,31 +6,37 @@ import it.pagopa.interop.purposeprocess.model._
 
 object PurposeConverter {
 
-  def dependencyToApi(purpose: DependencyPurpose): Purpose = Purpose(
-    id = purpose.id,
-    eserviceId = purpose.eserviceId,
-    consumerId = purpose.consumerId,
-    versions = purpose.versions.map(PurposeVersionConverter.dependencyToApi),
-    suspendedByConsumer = purpose.suspendedByConsumer,
-    suspendedByProducer = purpose.suspendedByProducer,
-    title = purpose.title,
-    description = purpose.description,
-    riskAnalysisForm = purpose.riskAnalysisForm.map(RiskAnalysisConverter.dependencyToApi),
-    createdAt = purpose.createdAt,
-    updatedAt = purpose.updatedAt
-  )
+  implicit class DependencyPurposeWrapper(private val purpose: DependencyPurpose) extends AnyVal {
+    def dependencyToApi(isRiskAnalysisValid: Boolean): Purpose = Purpose(
+      id = purpose.id,
+      eserviceId = purpose.eserviceId,
+      consumerId = purpose.consumerId,
+      versions = purpose.versions.map(PurposeVersionConverter.dependencyToApi),
+      suspendedByConsumer = purpose.suspendedByConsumer,
+      suspendedByProducer = purpose.suspendedByProducer,
+      title = purpose.title,
+      description = purpose.description,
+      riskAnalysisForm = purpose.riskAnalysisForm.map(RiskAnalysisConverter.dependencyToApi),
+      createdAt = purpose.createdAt,
+      updatedAt = purpose.updatedAt,
+      isRiskAnalysisValid = if (purpose.riskAnalysisForm.isDefined) isRiskAnalysisValid else false
+    )
+  }
 
-  def persistentToApi(purpose: PersistentPurpose): Purpose = Purpose(
-    id = purpose.id,
-    eserviceId = purpose.eserviceId,
-    consumerId = purpose.consumerId,
-    versions = purpose.versions.map(PurposeVersionConverter.persistentToApi),
-    suspendedByConsumer = purpose.suspendedByConsumer,
-    suspendedByProducer = purpose.suspendedByProducer,
-    title = purpose.title,
-    description = purpose.description,
-    riskAnalysisForm = purpose.riskAnalysisForm.map(RiskAnalysisConverter.persistentToApi),
-    createdAt = purpose.createdAt,
-    updatedAt = purpose.updatedAt
-  )
+  implicit class PersistentPurposeWrapper(private val purpose: PersistentPurpose) extends AnyVal {
+    def persistentToApi: Purpose = Purpose(
+      id = purpose.id,
+      eserviceId = purpose.eserviceId,
+      consumerId = purpose.consumerId,
+      versions = purpose.versions.map(PurposeVersionConverter.persistentToApi),
+      suspendedByConsumer = purpose.suspendedByConsumer,
+      suspendedByProducer = purpose.suspendedByProducer,
+      title = purpose.title,
+      description = purpose.description,
+      riskAnalysisForm = purpose.riskAnalysisForm.map(RiskAnalysisConverter.persistentToApi),
+      createdAt = purpose.createdAt,
+      updatedAt = purpose.updatedAt,
+      isRiskAnalysisValid = false
```

Comment:
I'm not totally happy to have a fake value here, but I see the point in removing a not required activity that may impact performances in listing.
If it's good for others it's fine by me, but I would ask you to add a comment here explaining why it has this implementation

Reply:
I agree with you these attribute has to be put outside Purpose and not inside